script/setup

```diff
@@ -22,6 +22,7 @@ echo "Cleaning out old logs"
 echo "Installing libraries and plugins"
 { gem list -i bundler || gem install bundler
   bundle
+  script/install_toxiproxy.sh
```

Comment:
This is here to encourage the use of it, but not all contributors will use `script/setup` or know to run toxiproxy in the background.